File: src/main/java/domain/Person1.java
package domain;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;

@Entity
public class Person1 {

    @Id
    private Long id;

    private String name;

    private Integer age;

    public Person1() {

    }

    public Person1(Long id, String name, Integer age) {
        this.id = id;
        this.name = name;
        this.age = age;
    }
}

File: src/main/java/entity/Person1.kt
package entity

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id
import jakarta.persistence.Table
import jakarta.persistence.Transient

@Entity
class Person1 (
    @Id
    private val id: Long = 0L,
    private var name: String = "",
    private var age: Int = -1,
)
